emitter.go

```diff
@@ -61,7 +61,8 @@ func CopyMulty(src io.Reader, writers ...io.Writer) (err error) {
 		if nr > 0 && len(buf) > nr {
 			payload := buf[:nr]
 			meta := payloadMeta(payload)
-			requestID := string(meta[1])
+	//		requestID := string(meta[1])
+			requestID := string(meta[0])
```

Comment:
I do not think this is the right fix. Since requestID always stored as the second index in meta array.
If you see panic on this line, it means that for some reason, meta is empty, which may indicate that input returned malformed record.

Reply:
But this change is already merged into master along with my other fix for kafka ip address list fix.